docs/reference/builder.md

```diff
@@ -226,8 +248,61 @@ following lines are all treated identically:
 
 The following parser directive is supported:
 
+* `syntax`
 * `escape`
 
+## syntax
+
+    # syntax=[remote image reference]
+
+For example:
+
+    # syntax=docker/dockerfile
+    # syntax=docker/dockerfile:1.0
+    # syntax=docker/dockerfile:1.0.0-experimental
+    # syntax=example.com/user/repo:tag@sha256:abcdef...
+
+This feature is only enabled if the [BuildKit](#buildkit) backend is used.
+
+The syntax directive defines the location of the Dockerfile builder that is used for
+building the current Dockerfile. The BuildKit backend allows to seamlessly use
+external implementations of builders that are distributed as Docker images and 
+execute inside a container sandbox environment.
+
+Using a custom Dockerfile implementation allows you to:
```

Comment:
"Using a" could be removed here;

```
Custom Dockerfile implementations allow you to:
```

Also, could you add a blank line between this line and the bullet-list?